cai_causal_graph/causal_graph.py

```diff
@@ -94,6 +94,11 @@ def __ne__(self, other: object) -> bool:
         """Check if the skeleton is not equal to another skeleton."""
         return not (self == other)
 
+    def __iter__(self) -> Iterator:
+        """Return item,value tuples of the content of self.to_dict()."""
```

Comment:
wrap self.to_dict in "`" for better formatting.

Reply:
Added on this and on the one for CausalGraph that was also missing :)